frontend/public/actions/features.ts

```diff
@@ -212,6 +223,7 @@ const ssarCheckActions = ssarChecks.map(({flag, resourceAttributes, after}) => {
 
 export const detectFeatures = () => (dispatch: Dispatch) => [
   detectOpenShift,
+  detectBaremetalPlatform,
```

Comment:
Please add a TODO comment above this line:

```ts
// TODO(vojtech): move this flag definition to metal3-plugin via ActionFeatureFlag extension
```

Reply:
Done